apps/aecontract/test/aecontract_SUITE.erl

```diff
@@ -556,6 +558,20 @@ init_per_testcase(TC, Config) when TC == sophia_aens_resolve;
     %% Disable name auction
     meck:expect(aec_governance, name_claim_bid_timeout, fun(_, _) -> 0 end),
     init_per_testcase_common(TC, Config);
+
+init_per_testcase(sophia_aens_update_transaction, Config) ->
+    ProtocolVsn = aec_hard_forks:protocol_vsn(?config(protocol, Config)),
+    case ProtocolVsn >= ?IRIS_PROTOCOL_VSN of
+        true ->
+            meck:expect(aec_governance, name_claim_bid_timeout, fun(_, _) -> 0 end),
+            init_per_testcase_common(sophia_aens_update_transaction,
+                                     [{sophia_version, ?SOPHIA_LIMA_FATE},
```

Comment:
I really hope you mean SOPHIA_IRIS_FATE?

Reply:
But also, these  (sophia/vm/abi_version) should already be set by `init_per_group` should not be necessary to set here.

Reply:
it's fixed now